core/src/main/resources/reference.conf

```diff
@@ -79,10 +79,10 @@ akka.persistence.cassandra {
   journal {
 
     # FQCN of the cassandra journal plugin
-    class = "akka.persistence.cassandra.journal.CassandraJournal"
+    class = "pekko.persistence.cassandra.journal.CassandraJournal"
```

Comment:
classes start with org.apache

Reply:
thanks for the hint. I just did the package rework. configuration is next.